src/guide/security/operational-security.md

```diff
@@ -0,0 +1,100 @@
+
+- Always keep your software updated to their latest version across all devices, including computers and phones. Regular updates help patch vulnerabilities and minimise potential risks associated with outdated software, even before such vulnerabilities are disclosed.
+
+- Develop a routine for periodically updating passwords and cryptographic keys. This proactive approach significantly contributes to enhancing overall security posture, since it is much harder to hit a moving target.
+
+## Using Browsers
+
+If an application connected to Iroha 2 features a web UI, your browser can either aid the security or pose a potential threat. It is essential to exercise caution, especially when it comes to the plugins you choose to install.
+
+Consider the following measures to enhance your browsing security:
+
+- Avoid using browsers with known bad security models. Specifically, the following:
+  - Chrome — proprietary.
+  - Safari — proprietary, build on the `WebKit` engine that is well known to have fundamental security problems.
+  - Opera — proprietary, has a history of leaking their users' data.
+  - Firefox — known to leak data and pre-install questionable extensions.
+  - Vivaldi — proprietary.
+  - Edge — all of the above.
+  - surf — limited in its features and is extremely outdated.
+
+- Give preference to browsers with solid history of valuing and protecting their users' privacy and security:
+  - [Librewolf](https://librewolf.net/), [Icecat](https://www.gnu.org/software/gnuzilla/), [Firedragon](https://github.com/dr460nf1r3/firedragon-browser), etc. — well established forks of Mozilla Firefox with added security features.
+  - [Ungoogled chromium](https://github.com/ungoogled-software/ungoogled-chromium) — a highly audited open-source version of Google Chrome that is enhanced with additional security measures and has all of the Google-related web services removed.
+  - [Brave](https://brave.com/) — a highly audited open-source version of [Google Chromium](https://www.chromium.org/Home/) that is enhanced with additional security measures; has a built-in <abbr title="Virtual Private Network">VPN</abbr> and ad blocker functionality.
+  - [Falkon](https://www.falkon.org/) — an open-source Qt-based web browser (built on `QtWebEngine`, a wrapper for [Google Chromium](https://www.chromium.org/Home/)) with known track record of being secure; has a number of extensions available for download from its [KDE store page](https://store.falkon.org/browse/).
+  - [Qutebrowser](https://qutebrowser.org/) — an open-source Qt-based web browser (built on `QtWebEngine`, a wrapper for [Google Chromium](https://www.chromium.org/Home/)) with known track record of being secure; has a unique keyboard-focused approach with minimalist GUI; considered to be a browser of choice for many security specialists.
+
+- Avoid enabling `JavaScript` unless necessary.
+
+- Use the browser's built-in confinement mechanism for plugins.
```

Comment:
> Use the browser's built-in confinement mechanism for plugins.

Can anyone (or @appetrosyan as the author) please elaborate on this part? What exactly is meant by it and how can we make it clearer for the reader?

My guess is: restricting the access rights that the installed plugins have, but I'd like some feedback.

Reply:
My guess is: restricting the access rights that the installed plugins have, but I'd like some feedback.

That's exactly it. Most plugins run all the time. Firefox allows you to disable plugins for private mode. 

Safari recently added profiles, where you can disable the plugins altogether for a specific profile.

Reply:
https://github.com/hyperledger/iroha-2-docs/pull/416/commits/e98d1b9ce9f5d6fedd5d86f71969407ad7996d84